ubuntu_cloud_image_changelog/ubuntu_cloud_image_changelog/lib.py

```diff
@@ -310,3 +326,8 @@ def get_changelog(
             )
 
     return cache_filename
+
+
+def sanitize_fips_version(package_version):
```

Comment:
Is this used anywhere now?

Reply:
Sorry, I updated the PR just now to remove it.